### DISCUSSION TITLE
Why remove the sticky note feature？

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#353] [Q&A]
Recently wanted to add a memo on Sticky, but. I couldn't find its whereabouts on Hello.
Update: I forgot to look carefully at the instructions for version 0.7.
But I don't know why I want to remove the sticky note.

**Comment by probonopd** [comment]
Project 'william.belanger/qtpad' was moved to 'william.belanger/primenote'. 
https://gitlab.com/william.belanger/primenote

I did not find the time to update
https://github.com/helloSystem/ISO/blob/09c5e293bbe55d79e1503fbeaf876ee0b0d79a86/settings/script.hello#L165-L180
yet.

Can you get it to work with the latest version again?

**Comment by louies0623** [comment]
Sorry, I don't understand the code, the only thing I know is that when there is only one person developing the software or system, if something happens to that person, the project will be very dangerous; in short, you need a team  to develop together.

**Reply by probonopd** [reply]
Yes! Volunteers?

**Reply by louies0623** [reply]
It really needs a few more volunteer developers to join, It does require more volunteer developers, like the one who wants to develop the hello store, and then assign the work to the right people, for example he can design the interface for the downloader.
